server/routes/api/attachments/attachments.ts

```diff
@@ -64,7 +50,7 @@ router.post(
     }
 
     const modelId = uuidv4();
-    const acl = AttachmentHelper.presetToAcl(preset);
+    const acl = AttachmentHelper.presetToAcl(preset!);
```

Comment:
Had to add the non-null assertion for `preset` because AFAIK it's optional if `public` is defined in the request. Not sure if there's a better to handle this?

Reply:
As commented `public` is deprecated and can be removed now, which simplifies things.

Reply:
Should I include that in this PR if it's a small change?

Reply:
Yea, might as well remove here as it'll make these types more accurate 👍

Reply:
Done. Hopefully I didn't miss something.